polyply/src/meta_molecule.py

```diff
@@ -87,6 +97,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.molecule = None
         nx.set_node_attributes(self, True, "build")
+        self.__search_tree = None
```

Comment:
`_search_tree` rather than `__search_tree` would have my preference

Reply:
We'll have a look when dealing with issue #158